src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -88,7 +88,11 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
     }
 
     if (shapeToolData?.supertypes?.behavior?.item?.schematicDesignation) {
-      setSchematicDesignation(shapeToolData?.supertypes?.behavior?.item?.schematicDesignation);
+      if (Array.isArray(shapeToolData?.supertypes?.behavior?.item?.schematicDesignation)) {
+        setSchematicDesignation(shapeToolData?.supertypes?.behavior?.item?.schematicDesignation.join(" "));
```

Comment:
@kostobog do we support multiple schematicDesignation, maybe we should join with ", "

Reply:
Should be resolve by: https://github.com/kbss-cvut/fta-fmea-ui/issues/361